darknet.py

```diff
@@ -215,313 +302,3 @@ def network_height(net):
-                rr, cc = draw.polygon_perimeter([x[1] for x in boundingBox], [x[0] for x in boundingBox], shape= shape)
-                rr2, cc2 = draw.polygon_perimeter([x[1] + 1 for x in boundingBox], [x[0] for x in boundingBox], shape= shape)
-                rr3, cc3 = draw.polygon_perimeter([x[1] - 1 for x in boundingBox], [x[0] for x in boundingBox], shape= shape)
-                rr4, cc4 = draw.polygon_perimeter([x[1] for x in boundingBox], [x[0] + 1 for x in boundingBox], shape= shape)
-                rr5, cc5 = draw.polygon_perimeter([x[1] for x in boundingBox], [x[0] - 1 for x in boundingBox], shape= shape)
-                boxColor = (int(255 * (1 - (confidence ** 2))), int(255 * (confidence ** 2)), 0)
-                draw.set_color(image, (rr, cc), boxColor, alpha= 0.8)
-                draw.set_color(image, (rr2, cc2), boxColor, alpha= 0.8)
-                draw.set_color(image, (rr3, cc3), boxColor, alpha= 0.8)
-                draw.set_color(image, (rr4, cc4), boxColor, alpha= 0.8)
-                draw.set_color(image, (rr5, cc5), boxColor, alpha= 0.8)
-            if not makeImageOnly:
-                io.imshow(image)
-                io.show()
-            detections = {
-                "detections": detections,
-                "image": image,
-                "caption": "\n<br/>".join(imcaption)
-            }
-        except Exception as e:
-            print("Unable to show image: "+str(e))
-    return detections
-
-def performBatchDetect(thresh= 0.25, configPath = "./cfg/yolov4.cfg", weightPath = "yolov4.weights", metaPath= "./cfg/coco.data", hier_thresh=.5, nms=.45, batch_size=3):
-    import cv2
-    import numpy as np
-    # NB! Image sizes should be the same
-    # You can change the images, yet, be sure that they have the same width and height
-    img_samples = ['data/person.jpg', 'data/person.jpg', 'data/person.jpg']
-    image_list = [cv2.imread(k) for k in img_samples]
-
-    net = load_net_custom(configPath.encode('utf-8'), weightPath.encode('utf-8'), 0, batch_size)
-    meta = load_meta(metaPath.encode('utf-8'))
-    pred_height, pred_width, c = image_list[0].shape
-    net_width, net_height = (network_width(net), network_height(net))
-    img_list = []
-    for custom_image_bgr in image_list:
-        custom_image = cv2.cvtColor(custom_image_bgr, cv2.COLOR_BGR2RGB)
-        custom_image = cv2.resize(
-            custom_image, (net_width, net_height), interpolation=cv2.INTER_NEAREST)
-        custom_image = custom_image.transpose(2, 0, 1)
-        img_list.append(custom_image)
-
-    arr = np.concatenate(img_list, axis=0)
-    arr = np.ascontiguousarray(arr.flat, dtype=np.float32) / 255.0
-    data = arr.ctypes.data_as(POINTER(c_float))
-    im = IMAGE(net_width, net_height, c, data)
-
-    batch_dets = network_predict_batch(net, im, batch_size, pred_width,
-                                                pred_height, thresh, hier_thresh, None, 0, 0)
```

Comment:
Can we add param `--batch 4` to use `network_predict_batch()` for batch-processing of video and images for `darknet_video.py` and `darknet_images.py`?